### PR TITLE
Release 0.6.0

### PR DESCRIPTION
**Pull request by bjosv**
### 0.6.0 - Feb 09, 2021
* Minimum required version of CMake changed to 3.11 (from 3.14)
* Re-added the Makefile for symmetry with hiredis, which also enables
  support for statically-linked libraries.
* Improved examples
* Corrected crashes and leaks in OOM scenarios
* New API for sending commands to specific node
* New API for node iteration, can be used for sending commands
  to some or all nodes.

### CODE REVIEW

File: command.c [hunk 1]
```diff
@@ -1,7 +1,7 @@
 #include <ctype.h>
 #include <errno.h>
 #include <hiredis/alloc.h>
-#include <string.h>
+#include <strings.h>
```
Comment (zuiderkwast) [inline command.c:4]: How could this have worked with the wring header? They don't define the same functions, e.g. string.h is for strcmp() while strings.s is for strcasecmp()...

**Review by zuiderkwast — COMMENTED** [review]
Looks good, just a question.

File: command.c (hunk 1 above)
Comment (bjosv) [reply command.c:4]: The strangest thing, its only triggered when building with makefiles aswell.
I see that CMake automatically sets `gnu99` when using the 99 standard,
while the makefile uses `c99` (as in hiredis and hiredis-vip) which also requires `_XOPEN_SOURCE=600`. 
This maybe trigger different indirect includes

Comment (zuiderkwast) [reply command.c:4]: As long as it works in all kinds of builds, I trust you.

**Pull request merged**